Guardfile.example

```diff
@@ -0,0 +1,72 @@
+# More info at https://github.com/guard/guard#readme
+
+# You'll need to make sure Guard, and any of its plugins are in your Gemfile.local
+#
+# Example:
+#  # Automatically run tests on file changes
+#  gem 'guard', require: false
+#  gem 'guard-rspec', require: false
```

Comment:
Not sure if I'm running this the expected way, but I had to add `gem 'guard-bundler`, require: false`, so that I could run `bundle exec guard`?

Reply:
You're running it right. I forgot that I left the bundler bit in the example config when I was copying things out of my Guardfile. I'll add that to the example.